src/VkBootstrap.h

```diff
@@ -131,6 +132,13 @@ enum class SwapchainError {
 	failed_get_swapchain_images,
 	failed_create_swapchain_image_views,
 };
+
+std::error_code make_error_code (InstanceError instance_error);
```

Comment:
This should be in the cpp file, since its purely an implementation helper.

Reply:
Making this in the cpp files will break the tests.

Reply:
Ah, okay, but can we put it in a detail namespace or something? I'm hoping to keep only whats necessary exposed directly in the vkb namespace.

Reply:
Ah, okay, but can we put it in a detail namespace or something? I'm hoping to keep only whats necessary exposed directly in the vkb namespace.

Reply:
That is an option. Though if the client-side cannot find `make_error_code` through ADL, it is not technically correct usage of `std::error_code`.

Reply:
If you think this needs to stay in the vkb namespace, I am not against it. I just feel is good to try to limit the api as much as possible.

Plus, if there is a reason to keep it in the vkb namespace, you don't have to make any changes, meaning I can merge his then be able to start working on documenting the change.

Reply:
Yeah, I think it provides some value the user can convert your enums into `std::error_code` themselves. Plus putting in another namespace would complicate the implementation (as `std::error_code` cannot find them automatically.)

Reply:
Sounds good then. 
I'll merge it shortly.